pkg/controller/bookkeepercluster/bookie.go

```diff
@@ -130,11 +140,11 @@ func makeBookiePodSpec(bk *v1alpha1.BookkeeperCluster) *corev1.PodSpec {
 				VolumeMounts: []corev1.VolumeMount{
 					{
 						Name:      LedgerDiskName,
-						MountPath: "/bk/ledgers",
+						MountPath: ledgerDirs,
 					},
 					{
 						Name:      JournalDiskName,
-						MountPath: "/bk/journal",
+						MountPath: journalDirs,
 					},
 					{
 						Name:      IndexDiskName,
```

Comment:
User may want to have a different mount path for IndexDirectories as wel.
Lets have the same check for IndexDirectories too...

Reply:
Done